### ISSUE TITLE
Doc:  What kind of network paths do we support?

### ISSUE THREAD

**Issue by ppf2** [elastic/enterprise-search-network-drives-connector#15]
Will be helpful to document the type of network drive paths we support, e.g. will it work against any network drives as long as the network paths are using local path, UNC, SMB, Windows mapped drivers, etc..

**Comment by izmaxxsun** [comment]
If it helps, I was able to set up this connector on some development VMs (running on GCP).  For the network file share I set up a Samba share on Debian 11 (I'm a newb on this so I followed this [tutorial](https://computingforgeeks.com/how-to-configure-samba-share-on-debian/)).  I totally messed up on specifying the path in the smb.conf by pointing to "/public" instead of where my files actually were "{HOME_DIR}/public" which got me hung up for a while.

Inside Workplace Search (I had self managed version 8.2.2 running), I didn't see "Custom Content Source" or the "Configure Network Drives Connector" option making me wonder if I missed a step somewhere but I forged on by picking "Custom API Source".

From there, it was a bunch of YML config change for the "network_drive_connector.yml" file which would be nicer if it's in a UI but I managed to eventually set all the options correctly.  The "enable_document_permission" defaulted to true so that threw a wrench in things until I realized I needed to turn it off. I think the error message said to check my user permission mappings so it took a minute to realize it meant the document permissions.
